docs/stats/robust.rst

```diff
@@ -0,0 +1,105 @@
+.. _stats-robust:
+
+******************************
+Robust Statistical Estimators
+******************************
+
+Robust statistics provide reliable estimates of basic statistics for complex
+distributions.  The statistics package includes several robust statistical 
+functions that are commonly used in astronomy.  This includes methods for 
+rejecting outliers as well as statistical description of the underlying 
```

Comment:
Something seems to be missing before "statistical description" -- a verb of some kind, e.g. "estimating [providing?] a statistical description ..."?